frontend/src/scenes/onboarding/sdks/SDKs.tsx

```diff
@@ -1,54 +1,92 @@
-import { LemonDivider, LemonInput, LemonSelect } from '@posthog/lemon-ui'
+import { LemonButton, LemonDivider, LemonSelect } from '@posthog/lemon-ui'
 import { sdksLogic } from './sdksLogic'
-import { LemonCard } from 'lib/lemon-ui/LemonCard/LemonCard'
 import { useActions, useValues } from 'kea'
 import { OnboardingStep } from '../OnboardingStep'
 import { SDKSnippet } from './SDKSnippet'
 import { onboardingLogic } from '../onboardingLogic'
+import { useEffect } from 'react'
+import React from 'react'
+import { SDKInstructionsMap } from '~/types'
 
-export function SDKs({ usersAction }: { usersAction?: string }): JSX.Element {
-    const { setSourceFilter, setSelectedSDK } = useActions(sdksLogic)
+export function SDKs({
+    usersAction,
+    sdkInstructionMap,
+}: {
+    usersAction?: string
+    sdkInstructionMap: SDKInstructionsMap
+}): JSX.Element {
+    const { setSourceFilter, setSelectedSDK, setAvailableSDKInstructionsMap } = useActions(sdksLogic)
     const { sourceFilter, sdks, selectedSDK, sourceOptions } = useValues(sdksLogic)
     const { productKey } = useValues(onboardingLogic)
 
+    useEffect(() => {
+        setAvailableSDKInstructionsMap(sdkInstructionMap)
+    }, [])
+
     return (
         <OnboardingStep
             title={`Where are you ${usersAction || 'collecting data'} from?`}
-            subtitle="Pick one to start and add more sources later."
+            subtitle="Pick one or two to start and add more sources later."
```

Comment:
I think the "sources" was a little vague to me. I think there is really only one source right now (Web) so maybe it's worth changing it to something like "Choose the framework your frontend is built on"